inst/private/python_ipc_pytave.m

```diff
@@ -59,17 +61,50 @@
 
   info.prelines = 0;
 
-  % TODO: bit wasteful to repeat this every call
-  headers = python_header();
-  pyexec(headers)
+  if isempty(have_headers)
+    pyexec(strjoin({'import sys',
+                    'import sympy',
+                    'import sympy as sp',
+                    'from sympy import __version__ as spver',
+                    'from sympy import *',
+                    'from sympy.logic.boolalg import Boolean, BooleanFunction',
+                    'from sympy.core.relational import Relational',
+                    '# temporary? for piecewise support',
+                    'from sympy.functions.elementary.piecewise import ExprCondPair',
+                    'from sympy.integrals.risch import NonElementaryIntegral',
+                    'from sympy.matrices.expressions.matexpr import MatrixElement',
+                    '# for hypergeometric',
+                    'from sympy.functions.special.hyper import TupleArg',
+                    'from sympy.utilities.iterables import uniq',
+                    'import copy',
+                    'import binascii',
+                    'import struct',
+                    'import codecs',
+                    'import xml.etree.ElementTree as ET',
+                    'from distutils.version import LooseVersion',
```

Comment:
Thanks. I did `git blame` but it only shows which changes in this file were made when/by whome.
But, some imports, although not required by `python_header.py`,are still required by some tests...
Since the imports are not that many, I'll try to remove them one by one and try.